mautrix_telegram/__main__.py

```diff
@@ -271,6 +284,54 @@ async def manhole_global_namespace(self, user_id: UserID) -> dict[str, Any]:
             "Puppet": Puppet,
         }
 
+    async def _notify_bridge_blocked(self, is_blocked: bool = True) -> None:
+        msg = self.config["bridge.limits.block_ends_notification"]
+        if is_blocked:
+            msg = self.config["bridge.limits.block_begins_notification"]
+            next_notification = self._last_blocking_notification + self.config["bridge.limits.block_notification_interval_seconds"]
+            # We're only checking if the block is active, since the block-end notification will not be resent and we want it ASAP
+            if next_notification > int(time()):
+                return
+            self._last_blocking_notification = int(time())
```

Comment:
I was checking that this had not `await` before it.
Looks good. :+1:

Edge case, with an interval of 0 and a burst of messages at the same time, a new admin may get invited to multiple alert root. But I think this isn't worth preventing.